iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -2127,4 +2127,26 @@ public List<FileStatus> getMergeTaskInputFiles(Properties properties) throws IOE
   public MergeTaskProperties getMergeTaskProperties(Properties properties) {
     return new IcebergMergeTaskProperties(properties);
   }
+
+  @Override
+  public boolean isIdentityPartitionSpec(org.apache.hadoop.hive.metastore.api.Table table,
```

Comment:
why should it be in API, could we move this to TableUtils?

Reply:
Done